modules/javafx.swing/src/main/java/javafx/embed/swing/package.html

```diff
@@ -37,11 +37,24 @@
 
 <body>
 <p>Provides the set of classes to use JavaFX inside Swing applications.</p>
```

Comment:
Unless you also remove this initial sentence, the docs are still misleading, since only the first sentence shows up in the module summary for a package.